README.md

```diff
@@ -212,8 +212,8 @@ These shortcuts are available in GUI mode:
 | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Z</kbd>                                            | Redo the next modification                    |
 | <kbd>Ctrl</kbd> + <kbd>Q</kbd>                                            | Leave the capture screen                                         |
 | <kbd>Ctrl</kbd> + <kbd>O</kbd>                                            | Choose an app to open the capture                                |
-| <kbd>Ctrl</kbd> + <kbd>Return</kbd>                                            | Commit text in text area|
-| <kbd>Return</kbd>                                             | Upload the selection to Imgur                                      |
+| <kbd>Ctrl</kbd> + <kbd>Enter</kbd>                                            | Commit text in text area|
+| <kbd>Enter</kbd>                                             | Upload the selection to Imgur                                      |
```

Comment:
The `Return` is also something that is used in the config file. I can see you point that on most keyboards it is written "Enter", and the only outlier is Mac, but having Return there is also to some extent educational (because of the config). Perhaps we should find a way to either accommodate both, or at least for now revert the change?

Reply:
Understood, reverted.👍